pkg/update/testdata/setters/expected/kustomization.yml

```diff
@@ -7,6 +7,7 @@ images:
   - name: container
     newName: index.repo.fake/updated # {"$imagepolicy": "automation-ns:policy:name"}
     newTag: v1.0.1 # {"$imagepolicy": "automation-ns:policy:tag"}
+    newDigest: sha256:6745aaad46d795c9836632e1fb62f24b7e7f4c843144da8e47a5465c411a14be # {"$imagepolicy": "automation-ns:digest:digest"}
```

Comment:
Shouldn't this be `# {"$imagepolicy": "automation-ns:policy:digest"}`? Or use some other name than digest for the policy, like this it is very confusing reading the tests.